docs/content/en/main/concepts/syncer.md

```diff
@@ -96,14 +96,40 @@ You can then create foo.bar in this workspace, or create an APIBinding in anothe
 To sync resource from another existing APIExport in the KCP server, run
 
 ```sh
-kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace|another-apiexport -o syncer.yaml
+kubectl kcp workload sync <mycluster> --syncer-image <image name> --apiexports another-workspace:another-apiexport -o syncer.yaml
 ```
 
 Syncer will start syncing the resources in this APIExport as long as the physical clusters has the compatible API schemas.
 
 To see if a certain resource is supported to be synced by the syncer, you can check the state of the `syncedResources` in `SyncTarget`
 status.
 
+### Bind workspace to the location workspace
+
+After the synctarget is ready, switch to the workspace (to deploy the workload later) and run
+
+```sh
+kubectl kcp bind compute <workspace of synctarget>
+```
+
+This command will create a `Placement` in the workspace and `APIBinding`s for global kubernetes `APIExport` and
```

Comment:
```suggestion
This command will create a `Placement` in the location workspace. By default it will also create `APIBinding`s for the global kubernetes `APIExport` and the
```

Reply:
placement is in this workspace, not location workspace

Reply:
You're right, sorry.